src/metalookup/core/content.py

```diff
@@ -182,52 +181,3 @@ async def soup(self) -> BeautifulSoup:
         if self._soup is None:
             self._soup = BeautifulSoup(await self.html(), "lxml")
         return self._soup
-
-    async def raw_links(self) -> list[str]:
```

Comment:
Also - it is used in malicious extensions - should that be updated/changed or are raw_links automatically created?

Reply:
I was hoping to fully eliminate the `raw_links` method from the `Content` class. 
Reasoning:
The set of links an extractor is interested in will _very likely_ depend on the context in which the link appears (e.g. `<a href="...">` vs `<link rel="stylesheet" href="...">`). Hence/IMHO it makes more sense to extract the set of relevant links within the respecvtive extractors as they know about their relevant contexts.
As it is still used in `MaliciousExtensions` right now, I would probably just move the `raw_links` implementation from the `Content` class to the `MaliciousExtensionsExtractor`. Then it can be refined there at some point if need be.

Reply:
Sounds good. Please do so.